cmd/broker/ingress/main.go

```diff
@@ -17,12 +17,17 @@ limitations under the License.
 package main
 
 import (
-	"github.com/google/knative-gcp/pkg/broker/ingress"
 	"go.uber.org/zap"
+
+	"github.com/google/knative-gcp/pkg/broker/ingress"
 	"knative.dev/pkg/logging"
 	"knative.dev/pkg/signals"
 )
 
+// main creates and starts an ingress handler using default options.
+// 1. it listens on port 8080
+// 2. it reads "GOOGLE_CLOUD_PROJECT" env var for pubsub project.
```

Comment:
I am expecting the controller to set this, since the controller needs this to talk to pubsub anyway. Let me know if this works for you.

Reply:
I was assuming that the controller creates the ingress but that may not be true. How is the controller going to get the project?

Reply:
For the pubsub channel API we have today, you can provide a project ID. If missing, I believe somewhere (either in our code or in the pubsub sdk) it will use the project ID in the cluster metadata (which is the project hosting the gke cluster).

Reply:
There is a `utils` package which reads project id from metadata. Going to use that.